docker/pcap-http-extractor/Dockerfile

```diff
@@ -0,0 +1,14 @@
+
+FROM demisto/python3-deb:1.0.0.53
+
+COPY requirements.txt .
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+  gcc \
+  python-dev \
+  tshark \
```

Comment:
Please install the thirdparty required debian packages in a separate RUN command as we do with: https://github.com/demisto/dockerfiles/blob/master/docker/office-utils/Dockerfile#L15

Reply:
I did it in #66a1394.
BTW, why is that needed?

Reply:
I added a small commit to also remove the lists. All looks good. Regarding your question it is not a really needed to separate but it makes things easier to understand with this separation, when there is need to review what the image contains.